### PR TITLE
Fix NoMethodError for Youtube Scraper

### PR DESCRIPTION
**Pull request by Spone**
see #180

### CODE REVIEW

File: lib/video_info/providers/youtube_scraper.rb [hunk 1]
```diff
@@ -64,7 +64,11 @@ def meta_nodes
 
       def meta_node_value(name)
         if available?
-          node = meta_nodes.detect { |n| n.attr('name').value == name }
+          node = meta_nodes.detect do |n|
+            n.attr('name').present? && n.attr('name').value == name
```
Comment (thibaudgg) [inline lib/video_info/providers/youtube_scraper.rb]: `.present?` is not a Ruby method and is only supported when `activesupport` gem is present, which is not a dependency of this gem, therefore we cannot using it here.

I would write it that way:
``` ruby
n.attr('name') && n.attr('name').value == name
```

File: lib/video_info/providers/youtube_scraper.rb [hunk 2]
```diff
@@ -64,7 +64,11 @@ def meta_nodes
...
+          end
+
+          return nil if node.attr('content').nil?
```
Comment (thibaudgg) [inline lib/video_info/providers/youtube_scraper.rb]: What about replacing the last two lines with:
``` ruby
node && node.attr('content') && node.attr('content').value
```

**Comment by Spone** [comment]
Hi @thibaudgg here is the spec 876fc7adc6a82f12d3b769979ffb2c3e7c27f1a1 and the syntax changes 7b434c21d61b5425e3407312fa8023e87cd632bc

**Comment by thibaudgg** [comment]
Looks good to me, thanks @Spone. 
@vheuken what do you think? I'm also seeing that TravisCI is broken, do you know what is happening here?

**Comment by vheuken** [comment]
@thibaudgg Regarding Travis-CI, looks like some of the specs have just code rotted a bit. Lots of stuff like:

```
  15) VideoInfo::Providers::Wistia with video pxonqr42is #duration should eq 250.0
      Failure/Error: it { is_expected.to eq 250.0 }
      
        expected: 250.0
             got: 250.428

  16) VideoInfo::Providers::Youtube with video WVsj2pS-zFY #description should eq nil
      Failure/Error: it { is_expected.to eq nil }
      
        expected: nil
             got: ""

  17) VideoInfo::Providers::Youtube with video mZqGqE0D0n4 #date should return date video was posted
      Failure/Error:
        is_expected.to eq Time.parse('Sat Apr 12 22:25:35 UTC 2008',
                                     Time.now.utc)
      
        expected: 2008-04-12 22:25:35.000000000 +0000
             got: 2008-04-12 22:34:48.000000000 +0000
      
        (compared using ==)
```

I think it's perfectly okay to merge in this code right now.

**Pull request merged**